### ISSUE TITLE
Wrong asset in release

### ISSUE THREAD

**Issue by brccabral** [geovens/gInk#183]
Hi,
The asset in 1.1.1 release has files from 1.1.0.
Thanks,
Bruno

**Comment by geovens** [comment]
Yeah I think I just added language files in the 1.1.1 asset, while reusing the unchanged executable files, so it still says "1.1.0" in the UI.
